virttest/utils_v2v.py

```diff
@@ -110,6 +123,54 @@ def get_cmd_options(self, params):
         self.new_name = self.params.get('new_name')
         self.net_vm_opts = ""
 
+        def _get_input_transport_options():
```

Comment:
get_input_transport_options change to compose_input_transport_xx ?

Reply:
Done